WcaOnRails/app/models/competition.rb

```diff
@@ -665,7 +671,13 @@ def refund_policy_percent_required?
   end
 
   def guests_entry_fee_required?
-    confirmed? && created_at.present? && created_at > Date.new(2018, 8, 22)
+    (
+      confirmed? && created_at.present? && created_at > Date.new(2018, 8, 22)
```

Comment:
Isn't a `&&` is missing at the end of this line?

Reply:
Oh damn! Good catch.